client/src/components/WorkflowInvocationState/WorkflowInvocationJob.vue

```diff
@@ -0,0 +1,38 @@
+<script setup lang="ts">
+import { faClock as faRegClock } from "@fortawesome/free-regular-svg-icons";
+import { faClock } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+
+import type { JobBaseModel } from "@/api/jobs";
+
+import { getJobDuration } from "../JobInformation/utilities";
+import { NON_TERMINAL_STATES } from "./util";
+
+import JobInformation from "../JobInformation/JobInformation.vue";
+import JobParameters from "../JobParameters/JobParameters.vue";
+import UtcDate from "../UtcDate.vue";
+
+const props = defineProps<{
+    job: JobBaseModel;
+}>();
+
+const jobDuration = getJobDuration(props.job) || null;
+</script>
+
+<template>
+    <div>
+        <div class="d-flex justify-content-between pb-1 pr-1">
+            <i>
+                <FontAwesomeIcon :icon="faRegClock" class="mr-1" />run
+                <UtcDate :date="props.job.create_time" mode="elapsed" />
```

Comment:
This isn't the duration you would think it is. Can you remove the duration ?

Reply:
So, this is the `create_time`; I thought it is the time that the job was created?
In this `<i>` we show Job was run 18 minutes ago (e.g.)

Are you still talking about this being the _entire time from scheduling to last to last update_?

Isn't that the time in the next `<i>`, conditionally rendered for `v-if="!NON_TERMINAL_STATES.includes(props.job.state)"`?

which itself is calculated as [galaxy/client/src/components/JobInformation/utilities.ts](https://github.com/galaxyproject/galaxy/blob/8f6c95536f8e484f85a736436f93d9cc4980a27a/client/src/components/JobInformation/utilities.ts#L1-L7):
https://github.com/galaxyproject/galaxy/blob/8f6c95536f8e484f85a736436f93d9cc4980a27a/client/src/components/JobInformation/utilities.ts#L1-L7 ?

For that one, I can look at the wallclock job metric instead?

Reply:
This is when the job was scheduled, that has nothing to do with what you'd call duration in a realistic production setup. duration would be wallclock time.

Reply:
I see.
 So should we remove either of these times from here?
If not, I understand the (terminal) one on the right is incorrect, and so do we still keep the one on the left?
And do we then place the job metric wallclock time here?

Reply:
So, I've removed both times from this view, and instead just simply used the `JobDetails` component that in turn uses the existing `JobMetrics` component in this latest commit https://github.com/galaxyproject/galaxy/pull/18615/commits/0c50efcb46763dd24c7e78db0c6d7c7967daffb9

https://github.com/user-attachments/assets/9aa39a53-6f78-40c4-8af4-129f1263bedd

Let me know if this works, since this is just a reuse of the existing job details view we see when we click on a `ContentItem`'s info option.